src/clean_air/models.py

```diff
@@ -37,7 +37,15 @@ class Metadata:
 
     @property
     def id(self):
-        return self.dataset_name.lower().replace(string.punctuation, "_")
+        id_str = self.name
+        for c in string.punctuation.replace("_", ""):
+            if c in id_str:
+                id_str = id_str.replace(c, "")
+        for c in string.whitespace:
+            if c in id_str:
+                id_str = id_str.replace(c, "_")
+        return id_str.lower()
```

Comment:
I don't really understand what this bit does but I'm assuming it's required and tested, and therefore looks fine to me.

Reply:
basically stripping out special characters and whitespace from the string we use as an ID so the result is more URL friendly.

(I'd prefer to use something like an integer for the ID, but without a central database issuing them, it's extra complexity to ensure they're unique. I could use a [GUID](http://guid.one/guid), but they're not terribly user friendly)

Reply:
Right, I see.  Well it seems fine for now at least.  Maybe we can think about integers or GUIDs at a later date when we have more systems in place, but doesn't seem necessary right now.  Good job!